README.md

```diff
@@ -1,116 +1,248 @@
+ 
+### History
 
-The handbook uses Python3. Set up the virtual environment, install
-requirements and copy the `.env` file to set environment variables.
+This repo replaces the previous version of the handbook, which can be found here: 
+https://github.com/opendata-swiss/ogd-handbook-site
 
-```
-python -m venv p3venv 
+## Technical setup
+
+The handbook is implemented with [Sphinx](https://www.sphinx-doc.org/en/master/), 
+Sphinx is a static site generator for documentations. 
+The source code is written in [reStructuredText](https://en.wikipedia.org/wiki/ReStructuredText). 
+
+The handbook is structured into:
+- The **Source**: as reStructuredText
+- The **Translations**: as string translations
+
+The **Source** can be found at `source/content` with some pages such as the homepage `source/index.rst`
+directly in the `source` directory. All source files have the ending `.rst` as they contain
+reStructuredText.
+Most source files are in German language, some are written in English.
+No matter what language of these files are in: German or English: the `.rst` files are considered together as 
+the German/source version of the handbook.
+
+The **Translations** can be found at `source/locale`. They contain text strings that have 
+been extracted from the **Source**. These text strings are stored along with translations in `.po` files.
+If no translation is given in these files: the fallback ist the orginal text in the **Source**.
+The tool used for translations is
+[sphinx-intl](https://www.sphinx-doc.org/en/master/usage/advanced/intl.html)
+
+**Locales**: Currently there are two locales of the handbook:
+- a German locale at `source/locale/de`: the translations there are left empty, 
+so that the fallback, the **Source** is used
+- a French locale at `source/locale/fr`: stores the translatable strings from the **Source** together with French translations for all German texts. English texts have not been translated.
+
+## Setup for local development
+
+The handbook can be downloaded and setup locally on your Computer.
+First open a terminal window and clone the repo:
+
+```commandline
+git clone git@github.com:opendata-swiss/ogdch-handbook.git
 cd ogdch-handbook
+```
+
+If you want to be able to build the handbook locally from its 
+**Source**, then you need to set up the virtual python3 environment and install the 
+requirements, as shown below:
+
+```
+python3 -m venv p3venv 
 source p3venv/bin/activate
+(p3venv) pip install wheel
 (p3venv) pip install -r requirements.txt
-(p3venv) cp .env.dist .env
 ```
 
-## Build the German Documentation
+Now everything is setup and you will be able to build the handbook locally and then view it 
+in your browser.
 
-The German documentation can be built locally like this:
+## Build the German locale
+
+The German locale is build from `source/locale/de` with the following command:
 
 ```
-(p3venv) cd docs
-(p3venv) make clean
 (p3venv) make -e SPHINXOPTS="-D language='de'" BUILDDIR="build/de" html
```

Comment:
I think it could be useful to say which directory they have to run this command from. Otherwise, "The German locale is built from `source/locale/de`" could make them think they have to navigate to `source/locale/de` before running it. (The same for building the French locale below.)